src/dashboard/Dashboard.js

```diff
@@ -1,13 +1,42 @@
 import React from 'react';
 import { Link } from 'react-router';
-import { Button } from 'reactstrap';
+import { Container, Jumbotron, Row, Col, Progress, Button } from 'reactstrap';
+import '../css/main.css';
```

Comment:
We'll still want to modify the CSS on top of this though, won't we? I mean, we don't want to spend lots of time, but it needs to at least feel intuitive, so it's likely we'll have other styles other than just the header banner.